playground/src/Editor/Editor.tsx

```diff
@@ -81,33 +83,9 @@ export default function Editor() {
     setSecondaryTool(tool);
   };
 
-  useEffect(() => {
-    async function initAsync() {
-      await init();
-      const response = await restore();
-      const [settingsSource, pythonSource] = response ?? [
-        stringify(Workspace.defaultSettings()),
-        DEFAULT_PYTHON_SOURCE,
-      ];
-
-      setSource({
-        revision: 0,
-        pythonSource,
-        settingsSource,
-      });
-      setRuffVersion(Workspace.version());
-    }
-
-    initAsync().catch(console.error);
-  }, []);
-
```

Comment:
@charliermarsh I think we showed an empty editor and the `initAsync` that loads the wasm and restores the python code raced against the user making any changes :laughing: